### ISSUE TITLE
Error trying to download from Madokami

### ISSUE THREAD

**Issue by ZedK** [riderkick/FMD#198]
I think Madokami changed the folder structure, now I can't list anything other than raws or autouploads. 

If I enter the manga address direct on manga info I can see the files but still can't download from it (it fails).

**Comment by riderkick** [comment]
Yes, I'm aware of that, now it has uncertain deep, which is hard to follow.
It's working fine here, give me the url!

**Comment by ZedK** [comment]
None is working for me. I'll try with a clean config/ini later.

Some links that I can list the files but can't download:
https://manga.madokami.com/Manga/J/JI/JIN/Jin
https://manga.madokami.com/Manga/O/ON/ONEP/Onepunch-Man
https://manga.madokami.com/Manga/O/OT/OTOY/Otoyomegatari

**Comment by ZedK** [comment]
Ah, and is it possible to just download the file without extracting and renaming it? In cases like onepunch man where the chapter numbering is unconventional I'm never sure of what got downloaded.
For example, it changes:
"Onepunch-Man - Ch.57.3(90).zip" to "0122 - - Ch(90)"
"Onepunch-Man - Ch.56.5 [SPECIAL] [mangalife].zip" to "0119 - - Ch [SPECIAL] [mangalife]"

**Comment by riderkick** [comment]
If you want download the zip file directly why not using the browser?
Check your "Save to" options, what is your manga folder name / chapter folder name?

**Comment by riderkick** [comment]
If you want to left chapter name unchanged, uncheck "remove manga name from chapter" and "chapter folder name" is `%CHAPTER%`

**Comment by riderkick** [comment]
There is a bug with madokami login, which is always show as valid when the account is invalid.
Maybe that's the reason your download always failed, because you input wrong user/password, but you don't know it, because fmd always show as valid/ok.

**Comment by ZedK** [comment]
> If you want to left chapter name unchanged, uncheck "remove manga name from chapter" and "chapter folder name" is %CHAPTER%

That works, thanks!

> There is a bug with madokami login, which is always show as valid when the account is invalid.
Maybe that's the reason your download always failed, because you input wrong user/password, but you don't know it, because fmd always show as valid/ok.

You're probably right, re-entered my user/pass and it worked with my previous installation.
